contexts/codemeta.jsonld

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Could you rename this file `codemeta-2.0.jsonld` and add a `codemeta-2.0.txt` (or `.md`) file alongside with the citation information and this link? https://archive.softwareheritage.org/swh:1:cnt:ecba88bee7bac07a81a6f45414f4f57ce5eb2d29;origin=https://github.com/codemeta/codemeta;visit=swh:1:snp:a35573dd8a59795b7d81055234352efd8b08a603;anchor=swh:1:rev:dbc58ffa8b088ae1b09ad335852d9767c402983d;path=/codemeta.jsonld

Reply:
Ok! What do you mean by `citation information` ?

Reply:
Sorry, I forgot the link: https://github.com/codemeta/codemeta?tab=readme-ov-file#codemeta-schema

Reply:
Thanks! :)